client/ayon_resolve/api/plugin.py

```diff
@@ -7,7 +8,8 @@
 
 from ayon_core.pipeline import (
     LoaderPlugin,
-    Creator as NewCreator
+    Creator as NewCreator,
```

Comment:
Can this be changed to be only `Creator`?

Reply:
Done !